service/test/controllers/usermanagement/UserControllerTest.java

```diff
@@ -98,23 +61,20 @@ public void testCreateUserWithoutContentType() {
   }
 
   @Test
-  public void testcreateUser() {
-    PowerMockito.mockStatic(RequestInterceptor.class);
-    when(RequestInterceptor.verifyRequestData(Mockito.anyObject()))
-        .thenReturn("{userId} uuiuhcf784508 8y8c79-fhh");
+  public void testCreateUserSuccess() {
     Map<String, Object> requestMap = new HashMap<>();
     Map<String, Object> innerMap = new HashMap<>();
     innerMap.put(JsonKey.PHONE_VERIFIED, true);
-    innerMap.put(JsonKey.PHONE, "8800088000");
+    innerMap.put(JsonKey.PHONE, phoneNumber);
     innerMap.put(JsonKey.COUNTRY_CODE, "+91");
-    innerMap.put(JsonKey.EMAIL, "abbc@gmail.com");
-    innerMap.put(JsonKey.USERNAME, "userName");
-    innerMap.put(JsonKey.FIRST_NAME, "john");
-    innerMap.put(JsonKey.LAST_NAME, "rambo");
+    innerMap.put(JsonKey.EMAIL, emailId);
+    innerMap.put(JsonKey.USERNAME, userName);
+    innerMap.put(JsonKey.FIRST_NAME, firstName);
+    innerMap.put(JsonKey.LAST_NAME, lastName);
     List<String> roles = new ArrayList<>();
-    roles.add("user");
+    roles.add(role);
     List languages = new ArrayList<>();
-    languages.add("hindi");
+    languages.add(language);
 
```

Comment:
Result result = route(req);
     assertEquals(200, result.status();// Match response status and status-code as well